cognite/neat/rules/models/_base_rules.py

```diff
@@ -281,46 +181,69 @@ def dump(
             exclude_defaults=exclude_defaults,
         )
 
+    @classmethod
+    def headers_by_sheet(cls, by_alias: bool = False) -> dict[str, list[str]]:
+        """Returns a list of headers for the model."""
```

Comment:
I need a bit of context where it is used
```suggestion
        """Returns a list of headers for the model, typically used by ExcelExporter"""
```